lib/libutee/arch/arm/utee_syscalls_a32.S

```diff
@@ -23,6 +23,16 @@
 UNWIND( .fnstart)
         push    {r5-r7,lr}
 UNWIND( .save   {r5-r7,lr})
+#if defined(CFG_TA_FTRACE_SUPPORT) && !defined(__LDELF__)
```

Comment:
Shouldn't that be `CFG_ULIBS_MCOUNT`? (and same below for a64)

Reply:
Yeah that should make it work for `GPROF` too. Will use `CFG_ULIBS_MCOUNT` instead.